monitoring_prometheus/models/psutils_helpers.py

```diff
@@ -0,0 +1,33 @@
+import psutil
+from prometheus_client import Gauge
+
+MEMORY_USAGE_VMS = Gauge("memory_user_vms_mb", "Memory usage in MB", ["process", "pid"])
+
+MEMORY_USAGE_RSS = Gauge("memory_user_rss_mb", "Memory usage in MB", ["process", "pid"])
+
+
+def get_process_info():
+    for process in psutil.process_iter(
+        ["pid", "name", "memory_full_info", "cmdline", "nice"]
+    ):
+        try:
+            if process.info["memory_full_info"]:
+                if process.info["nice"] == 10:
+                    ProcessLabel = "workercron"
+                elif process.info["pid"] == 1:
+                    ProcessLabel = "dispatcher"
+                elif any(["gevent" in x for x in process.info["cmdline"]]):
+                    ProcessLabel = "gevent"
+                elif any(["odoo" in x for x in process.info["cmdline"]]):
```

Comment:
```suggestion
                elif any("odoo" in x for x in process.info["cmdline"]):
```

stops at the first that match